builtin/providers/aws/resource_aws_elb.go

```diff
@@ -121,6 +121,35 @@ func resourceAwsElb() *schema.Resource {
 				Default:  300,
 			},
 
+			"access_logs": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
+							Default:  false,
+						},
+						"interval": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default:  60,
+						},
+						"bucket": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+							Default:  "",
+						},
+						"bucket_prefix": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "",
```

Comment:
AWS handles this situation is like this:
- if you only have the ` bucket name ` (with or without the creation option) then the logs will be put here in ` <bucket-name>/AWSLogs/<account-id>/elasticloadbalancing/<region>/<year>/<month>/<day>/ `
- if you have the ` bucket name / prefix ` then the logs will be stored under ` <bucket-name>/<prefix>/AWSLogs/<account-id>/elasticloadbalancing/<region>/<year>/<month>/<day>/ `

Now, the issue would be how should Terraform handle this (and how does the AWS sdk does it as well)? If we use the `Computed` option then the default value should still be `""` but if we use an existing prefix, we should make sure that the default parts are removed from the output.